engine/src/main/java/org/terasology/persistence/internal/ReadWriteStorageManager.java

```diff
@@ -439,6 +443,8 @@ private void startAutoSaving() {
         saveTransaction = createSaveTransaction();
         saveThreadManager.offer(saveTransaction);
 
+        saveGamePreviewImage();
```

Comment:
IMHO this is too frequent to do automatic screenshots. Autosave is IIRC every 30 seconds? That's 120 an hour, or about 8 hours of gameplay to get to a thousand screenshots, which at an average size of 100kb would be 100 MB of screenshots :-)

I'd stick to generating an automatic screenshot on every manual save (exiting the world). Optionally we could add something under player settings to automatically take screenshots every x but I'm not convinced there'd really be a demand for that.